cylc/flow/etc/examples/converging-workflow/flow.cylc

```diff
@@ -0,0 +1,38 @@
+[meta]
+    title = Converging Workflow
+    description = """
+        A workflow which runs a pattern of tasks over and over until a
+        convergence condition has been met.
+    """
+
+[scheduling]
+    cycling mode = integer
+    initial cycle point = 1
+    [[graph]]
+        P1 = """
+            # run "increment" then check whether the convergence condition has
+            # been met
+            increment => check_convergence
+
+            # if the workflow has not converged yet, then run another cycle
+            check_convergence:not_converged? => increment[+P1]
+
+            # if the workflow has converged, then do nothing
+            check_convergence:converged?
+        """
+
+[runtime]
+    [[increment]]
+        # a task which evolves the data
+    [[check_convergence]]
+        # a task which checks whether the convergence condition has been met
+        script = """
+            if [[ ]]; then
```

Comment:
This might be sufficient?
```suggestion
            if (( CYLC_TASK_CYCLE_POINT == 4 )); then
                # for the purpose of example, assume convergence at cycle point 4
```